config.xml

```diff
@@ -8,6 +8,7 @@
       Red Hat Mobile
   </author>
   <content src="index.html"/>
+  <access origin="*"/>
   <access origin="https://fonts.googleapis.com"/>
```

Comment:
<access origin="*"/> allows all origins in this case you dont need to be specific and you can use only `*`
to be more specific you need use something like <access origin="https://*.feedhenry.com"/>